docs-ref-services/storage.md

```diff
@@ -16,19 +15,79 @@ The Azure Storage libraries for Java provide classes for working with data in yo
-| Version 12 (preview) | Blob, Queue, and File | [Blob](https://search.maven.org/artifact/com.azure/azure-storage-blob/)<br />[Queue](https://search.maven.org/artifact/com.azure/azure-storage-queue/)<br />[File](https://search.maven.org/artifact/com.azure/azure-storage-file/) | [Version 12 reference](https://docs.microsoft.com/java/api/overview/azure/storage?view=azure-java-preview) | [Blob](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/storage/azure-storage-blob)<br />[Queue](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/storage/azure-storage-queue)<br />[File](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/storage/azure-storage-file) |
-| Version 8 | Blob, Queue, File, and Table | [All services](https://mvnrepository.com/artifact/com.microsoft.azure/azure-storage) | [Version 8 reference](https://docs.microsoft.com/java/api/overview/azure/storage/client?view=azure-java-stable) | [All services](https://github.com/azure/azure-storage-java/tree/legacy-master) |
+| Version 12 | Blob and Queue | [Blob](https://search.maven.org/artifact/com.azure/azure-storage-blob/)<br />[Queue](https://search.maven.org/artifact/com.azure/azure-storage-queue/) | [Blob](https://azuresdkdocs.blob.core.windows.net/$web/java/azure-storage-blob/12.0.0/index.html)<br/>[Queue](https://azuresdkdocs.blob.core.windows.net/$web/java/azure-storage-queue/12.0.0/index.html) | [Blob](https://github.com/Azure/azure-sdk-for-java/tree/azure-storage-blob_12.0.0/sdk/storage/azure-storage-blob) ([Quickstart](/azure/storage/blobs/storage-quickstart-blobs-java))<br />[Queue](https://github.com/Azure/azure-sdk-for-java/tree/azure-storage-blob_12.0.0/sdk/storage/azure-storage-queue) |
+| Version 8 | Blob, Queue, File, and Table | [All services](https://mvnrepository.com/artifact/com.microsoft.azure/azure-storage) | [Version 8 reference](https://docs.microsoft.com/java/api/overview/azure/storage/client?view=azure-java-stable) | [All services](https://github.com/azure/azure-storage-java/tree/legacy-master) ([Quickstart](/azure/storage/blobs/storage-quickstart-blobs-java-legacy)) |
+
+### Accessing other storage account services with v12 libraries
 
-To use the client library in your project, add a dependency to your Maven `pom.xml` file. To learn how to add the dependency, visit the source repository listed in the table above for the client library you wish to use. For more information about adding a dependency in Java, see [Add a dependency](https://maven.apache.org/guides/getting-started/index.html#How_do_I_use_external_dependencies).
+Currently, the following libraries are in public preview:
 
-For more information about using the Azure Storage client library for Java, see the following articles:
+- File shares
+- File (Data lake)
```

Comment:
One more comment from the storage team - let's just call data lake Data Lake without File